ConsoleApp1/Lesson28_Queries.sql

```diff
@@ -0,0 +1,55 @@
+GO
+CREATE TABLE dbo.PhoneBook
+    (
+      Id INT IDENTITY(1, 1) NOT NULL PRIMARY KEY,
+      FirstName VARCHAR(64) NOT NULL,
+	  MiddleName VARCHAR(64) NULL,
+      LastName VARCHAR(64) NULL,
+	  phone VARCHAR NOT NULL,
```

Comment:
What if the same Person has more than 1 phone number?
How did you decide what should be the primary key?